workflow-templates/im-deploy-tf-manual-apply.yml

```
@@ -14,18 +14,18 @@
 #    - Set up a deployment board if it does not already exist: https://github.com/im-practices/git-er-done/blob/main/actions/deployment-board.md
 
 name: Apply Terraform
-run-name: Apply ${{ inputs.branch-tag-sha }} terraform to ${{ inputs.environment-or-target }}
+run-name: Apply ${{ inputs.root-module }} terraform at ${{ inputs.branch-tag-sha }}
 on:
   workflow_dispatch:
     inputs:
       branch-tag-sha:
         description: The branch, tag or sha of the terraform that should be deployed.  For Production, only tags reachable by the default branch will be accepted.
         required: true
-      environment-or-target:
-        description: The environment or secondary region target to deploy to
+      root-module:
```

```suggestion
      root-module: # This is used in loading environments secrets and was previously called environment
```

I think we need a little bit of a description on this new input.